node/src/tasks/bsp_submit_proof.rs

```diff
@@ -161,6 +161,8 @@ where
                 .await
                 .ok_or_else(|| anyhow!("Failed to get forest storage."))?;
 
+            info!(target: LOG_TARGET, "Forest root of BSP: {:?}", fs.read().await.root());
```

Comment:
Do you intend these logs to stay here? Wouldn't it be too much maybe

Reply:
Ups, leftovers from debugging. I'll delete them